tests/pkcli/test2_data/2.in/subprocess_test.py

```diff
@@ -0,0 +1,8 @@
+import sys
```

Comment:
There's already a test file. I'm a little confused why we need two tests that test the same thing.

Reply:
Good point. I'll try to share